SDK/simVis/View.cpp

```diff
@@ -868,23 +868,31 @@ unsigned int View::getNumInsets() const
 
 int View::getIndexOfInset(simVis::View* view) const
 {
-  InsetViews::const_iterator iter = std::find(insets_.begin(), insets_.end(), static_cast<osgViewer::View*>(view));
```

Comment:
I understand the need to change this, because std::find won't be happy.  We generally have a rule to remove dead code unless there's a strong reason to keep; recommend removing the commented section.

Reply:
I went ahead and updated this myself in the PR, I hope that's not bad etiquette.  I'm downloading the diff now to double check before merging.  Thanks for taking a look at this.  I can't guarantee we won't miss others since we're not building with OSG_USE_REF_PTR_IMPLICIT_OUTPUT_CONVERSION=OFF but this is a great start and a big help.

Reply:
No bad etiquette at all :) I left the commented-out code in there just as a reference in case I made a mistake.